### PR TITLE
Added annotation to the quantified expression

### PR DESCRIPTION
**Pull request by yuanzexiong**
# Pull Request

## Description

Added annotation to the quantified expression

Partially Closes #58 

## Changes

1. New function that handles quantified expression annotation
  - It returns just a simple Boolean
2. Added new switch case to trigger the function

## Deletes Source Branch?

Yes:    Work is done, the branch can be safely deleted.

## How Many Approvals?

- 1 Approval (Small bugs/hot-fixes)

### CODE REVIEW

**Review by HectorPeeters — COMMENTED** [review]
Does the quantified expression contain any child nodes? Because then you will have to annotate them as well and add them to compileAstToExpression.ts.

**Comment by yuanzexiong** [comment]
> Does the quantified expression contain any child nodes? Because then you will have to annotate them as well and add them to compileAstToExpression.ts.

Yes it absolutely does but some is of the types that we cannot resolve yet at the moment. Things like `varName`, ` quantifiedExprInClause`

File: src/typeInference/annotateAST.ts [hunk 1]
```diff
@@ -267,6 +268,13 @@ export function annotate(ast: IAST, staticContext: StaticContext): SequenceType
 			);
 			annotateTypeSwitchOperator(ast);
 
+		case 'quantifiedExpr':
+			// Variable amount of children from this AST node, solution: looping through them
+			for (let iterationCounter = 1; iterationCounter < ast.length; iterationCounter++) {
```
Comment (FBroy) [inline src/typeInference/annotateAST.ts]: Maybe you could use astHelper.getChildren(ast, '\*') for this one and then loop with a map. That would maybe be cleaner.
astHelper.getChildren(ast, '\*').map((a) => annotate(a, staticContext));

Comment (yuanzexiong) [reply src/typeInference/annotateAST.ts]: Thanks that definitely looks a lot better indeed

**Review by FBroy — APPROVED** [review]
Ready to be merged when the pipeline passes!

**Review by HectorPeeters — APPROVED** [review]

**Review by MTimmerije — APPROVED** [review]
Looks good to me, if it passes the pipeline, it can be merged.

**Pull request merged**